zowe-cli-id-federation-plugin/__tests__/__system__/cli/map/__snapshots__/cli.map.system.test.ts.snap

```diff
@@ -30,17 +31,17 @@ exports[`id-federation map should display the help 1`] = `
 
    --esm | -e (string)
 
-      The ESM to execute command
+      The ESM product on the target system
 
       Allowed values: RACF, TSS, ACF2
 
-   --lpar | -l (string)
+   --system | -l (string)
 
-      The security domain on which command will be executed
+      The target JES system on which the command will be executed
 
    --registry | -r (string)
 
-      The registry that contains the distributed-identity user name
+      The distributed identities registry
```

Comment:
could it show what it looks like? Is it an URL for example?

Reply:
I am not sure about all possibilities. It can be probably any string that groups identities together. But most likely it will be some URL — for example ldap://1.2.3.4 or enterprise.okta.com.

I don't know how to add an example. Should I add it directly to the description?

Reply:
I was thinking about adding them to the description, yes

Reply:
I've used the hostname of our okta server for the registry name in the past (like: broadcom.okta.com), but it could be really anything. The important fact here is, that it should be the same value for all the mappings which will be queried from one Zowe instance.
For the beginning, we plan that the registry name will be configurable param of Gateway so all users of that Gateway would need to have the mapping with this one registry name.